huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_attachments_test.go

```diff
@@ -0,0 +1,194 @@
+package cc
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceCcCentralNetworkAttachments_basic(t *testing.T) {
+	dataSource := "data.huaweicloud_cc_central_network_attachments.filter_by_id"
+	rName := acceptance.RandomAccResourceName()
+	dc := acceptance.InitDataSourceCheck(dataSource)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckCCProjectID(t)
+			acceptance.TestAccPreCheckCCRegionName(t)
+			acceptance.TestAccPreCheckCCGlobalGateway(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceCcCentralNetworkAttachments_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.name"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.central_network_id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.enterprise_router_id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.enterprise_router_project_id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_attachments.0.enterprise_router_region_id"),
+					resource.TestCheckOutput("is_id_useful", "true"),
+					resource.TestCheckOutput("is_name_useful", "true"),
+					resource.TestCheckOutput("is_type_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testDataSourceCcCentralNetworkAttachments_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+locals {
+  id   = huaweicloud_cc_central_network_attachment.test.id
+  name = huaweicloud_cc_central_network_attachment.test.name
+  type = "GDGW"
+}
+
+data "huaweicloud_cc_central_network_attachments" "filter_by_id" {
+  central_network_id = huaweicloud_cc_central_network.test.id
+  attachment_id      = local.id
+
+  depends_on = [
+    huaweicloud_cc_central_network_attachment.test,
+  ]
+}
+
+output "is_id_useful" {
```

Comment:
```suggestion
output "is_id_filter_useful" {
```
and fix other places

Reply:
fixed